tendermint/tests/lite.rs

```diff
@@ -125,6 +198,57 @@ fn run_test_cases(cases: TestCases) {
     }
 }
 
+#[test]
+fn bisection_simple() {
+    let case: TestBisection =
+        serde_json::from_str(&read_json_fixture("many_header_bisection/happy_path")).unwrap();
```

Comment:
@Liamsi 
TODO: fix needed here. Failing to unwrap with error "missing field `height`" at line 7 col 3

Reply:
Will fix it. I get a slightly different error (not related to height) though:
```
called `Result::unwrap()` on an `Err` value: Error("parse error: bad encoding", line: 7, column: 3)
thread 'bisection_simple' panicked at 'called `Result::unwrap()` on an `Err` value: Error("parse error: bad encoding", line: 7, column: 3)', src/libcore/result.rs:1165:5
```

Reply:
The reason was a typo in height: https://github.com/interchainio/tendermint-rs/pull/146/commits/8f78bbd0ce3a9cf2dbcaf97b7503d7abe9c901bb
not sure if this stems from the generator or was changed manually.

Reply:
Fixed in 8f78bbd0ce3a9cf2dbcaf97b7503d7abe9c901bb

Reply:
That was a typo in the json tag in generator. Thanks!